examples/qaoa.py

```diff
@@ -0,0 +1,166 @@
+  │           │           │           │           │           │
+  H           H           H           H           H           H
+  │           │           │           │           │           │
+  ZZ──────────ZZ^(-4/13)  │           │           │           │
+┌ │           │           │           │           │           │           ┐
+│ ZZ──────────┼───────────ZZ^(-4/13)  │           │           │           │
+│ │           ZZ──────────┼───────────ZZ^(-4/13)  │           │           │
+└ │           │           │           │           │           │           ┘
+┌ │           │           │           │           │           │           ┐
+│ ZZ──────────┼───────────┼───────────┼───────────ZZ^(-4/13)  │           │
+│ │           ZZ──────────┼───────────┼───────────┼───────────ZZ^(-4/13)  │
+└ │           │           │           │           │           │           ┘
+  Rx(0.151π)  Rx(0.151π)  ZZ──────────┼───────────ZZ^(-4/13)  │
+  │           │           │           │           │           │
+  ZZ──────────ZZ^-0.941   ZZ──────────┼───────────┼───────────ZZ^(-4/13)
+  │           │           │           ZZ──────────ZZ^(-4/13)  │
+┌ │           │           │           │           │           │           ┐
+│ │           │           Rx(0.151π)  ZZ──────────┼───────────ZZ^(-4/13)  │
+│ │           │           │           │           Rx(0.151π)  │           │
+└ │           │           │           │           │           │           ┘
+  ZZ──────────┼───────────ZZ^-0.941   Rx(0.151π)  │           Rx(0.151π)
+┌ │           │           │           │           │           │           ┐
+│ ZZ──────────┼───────────┼───────────┼───────────ZZ^-0.941   │           │
+│ │           ZZ──────────┼───────────ZZ^-0.941   │           │           │
+└ │           │           │           │           │           │           ┘
+  Rx(-0.448π) ZZ──────────┼───────────┼───────────┼───────────ZZ^-0.941
+  │           │           ZZ──────────┼───────────ZZ^-0.941   │
+  │           │           │           │           │           │
+  │           Rx(-0.448π) ZZ──────────┼───────────┼───────────ZZ^-0.941
+  │           │           │           ZZ──────────ZZ^-0.941   │
+┌ │           │           │           │           │           │           ┐
+│ │           │           Rx(-0.448π) ZZ──────────┼───────────ZZ^-0.941   │
+│ │           │           │           │           Rx(-0.448π) │           │
+└ │           │           │           │           │           │           ┘
+  │           │           │           Rx(-0.448π) │           Rx(-0.448π)
+  │           │           │           │           │           │
+  M('m')──────M───────────M───────────M───────────M───────────M
+  │           │           │           │           │           │
+Optimizing objective function ...
+The largest cut value found was 7.
+The largest possible cut has size 7.
+The approximation ratio achieved is 1.0.
+"""
+
+import itertools
+
+import networkx
+import numpy as np
```

Comment:
examples/qaoa.py:54:0: C0411: third party import "import numpy as np" should be placed before "import networkx" (wrong-import-order)

Reply:
Done